lnwire/funding_locked.go

```diff
@@ -0,0 +1,151 @@
+package lnwire
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/roasbeef/btcd/btcec"
+	"github.com/roasbeef/btcd/wire"
+)
+
+type FundingLocked struct {
+	ChannelOutpoint *wire.OutPoint
```

Comment:
Isn't this supposed to be the temporary channel id used throughout the funding process? https://github.com/lightningnetwork/lightning-rfc/blob/master/02-peer-protocol.md#the-funding_locked-message